requirements.txt

```diff
@@ -7,32 +7,33 @@
 aniso8601==7.0.0          # via graphene
 certifi==2019.9.11        # via requests, sentry-sdk
 chardet==3.0.4            # via requests
-coreapi==2.3.3
+coreapi==2.3.3            # via -r requirements.in
 coreschema==0.0.4         # via coreapi
-django-anymail==7.0.0
-django-cors-headers==3.1.1
-django-enumfields==1.0.0
-django-environ==0.4.5
-django-filter==2.2.0
-django-graphql-geojson==0.1.4
-django-graphql-jwt==0.2.2
-django-helusers==0.5.2
-django-ilmoitin==0.2.0
+django-anymail==7.0.0     # via -r requirements.in, django-ilmoitin
+django-cors-headers==3.1.1  # via -r requirements.in
```

Comment:
how did you get all of these `# via -r requirements.in`? 🤔

Reply:
Just by running `pip-compile requirements.in`. I had some trouble with the whole `venv` seutp, so I had to run it with the absolute path from the venv bin directory, so idk if that could've affected somehow...

If you remove the `requirements.txt` and run `pip-compile`, do you get it with or without those comments? Maybe it's a newer version?

Reply:
hm, interesting. I also get now these `via -r requirements.in`. I guess it's the way it should be with the primary dependencies now. Change was apparently introduced in [pip-tools v4.5.0](https://github.com/jazzband/pip-tools/blob/4.5.1/CHANGELOG.md#450-2020-02-20)

Reply:
Good catch, I kinda like it better, it gives you a bit more `T - Transparency` on what comes from where.